distributed/comm/ucx.py

```diff
@@ -158,13 +158,9 @@ def peer_address(self) -> str:
                 )
 
                 # Send meta data
+                cuda_frames = [hasattr(f, "__cuda_array_interface__") for f in frames]
                 await self.ep.send(np.array([len(frames)], dtype=np.uint64))
-                await self.ep.send(
-                    np.array(
-                        [hasattr(f, "__cuda_array_interface__") for f in frames],
-                        dtype=np.bool,
-                    )
-                )
+                await self.ep.send(np.array(cuda_frames, dtype=np.bool))
```

Comment:
We can then just send it as-is.

```suggestion
                await self.ep.send(cuda_frames)
```